COTX-Gateways

FrankZhaoX opened the pull request:
<!--
Thanks for submitting a pull request. Please fill the template below,
otherwise we will not be able to process this pull request.
-->

#### Summary
<!--
A short summary, referencing related issues:
Closes #0000, References #0000, etc.
-->

...

#### Screenshots
<!-- Post a screenshot of your rendered content
NOTE: This section is optional.
-->

...

#### Changes
<!-- What are the changes made in this pull request? -->

- ...
- ...

#### Notes for Reviewers
<!--
NOTE: This section is optional.

Motivate briefly why it is implemented this way, if that deviates from the
implementation proposal in the referenced issues.
- How should your reviewers approach this pull request?
- @mention reviewers with special requests or questions for them
-->

...

#### Checklist
<!-- Make sure that this pull request is complete. -->

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

FrankZhaoX committed:
Create COTX-Gateways_index.md

FrankZhaoX committed:
Merge pull request #1 from FrankZhaoX/COTX

Create COTX-Gateways_index.md

FrankZhaoX committed:
Create COTX-X1 Lite gateway

FrankZhaoX committed:
Add files via upload

FrankZhaoX committed:
Update COTX-X1 Lite gateway

FrankZhaoX committed:
Update COTX-X1 Lite gateway

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/TheThingsNetwork/docs?pullRequest=519) <br/>All committers have signed the CLA.

johanstokking reviewed (changes requested):
Thanks for your contribution.

The files will not render correctly in our documentation. Please see other gateway content pages for reference. Some issues:

- Missing frontmatter (title)
- Header formats (`#`) instead of links
- No references to the images using the `figure` tag
- Inconsistent spacing
- Wrong usage of "TTN" (use `{{% tts %}}` to refer to The Things Stack)
- Do not assume that the region is Europe; it can be any region of the user's choice

Please see https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/gateways/thethingsindoorgateway/_index.md for reference.

FrankZhaoX committed:
Update COTX-X1 Lite gateway

1. add title
2. "no wrap" to "soft wrap"
3. change the image reference to, for example, "![Lite-1.png](./TTN-X1%20Lite-1.png)"
4. change the ttn to {{% tts %}}. I've not change the titles, only one change in "1.Create a new {{% tts %}} account or login to your previously created {{% tts %}} account."

wait to change or ask
1. "8.The default connection of Gateway is Europe1, Gateway Server Address is: eu1.cloud.thethings.network, no additional other configuration. "
2. "Inconsistent spacing". I've not previewed

nejraselimovic commented on doc/content/gateways/COTX-Gateways_index.md:
```diff
@@ -0,0 +1,12 @@
+##COTX Gateways
+
```
```suggestion
---
title: "COTX Gateways"
---
```

nejraselimovic commented on doc/content/gateways/COTX-Gateways_index.md:
```diff
@@ -0,0 +1,12 @@
+
+COTX product portfolio consists of IoT modules, LoRa gateways, node devices and all-in-one solutions. We have over 80 varieties of products in active production. 
+They offer high quality, outstanding radio performances and carrier-grade security, and support US915, AU915, AS923, EU868, IN865, RU867, CN470 etc. 
+Please consult https://www.cotxnetworks.com/ for the full product list.                                                                                                                                                                                                                                                                                                                     
```
```suggestion
[COTX product portfolio](https://www.cotxnetworks.com/) consists of IoT modules, LoRa gateways, node devices and all-in-one solutions. COTX has over 80 types of products in active production. They offer high quality radio performances and carrier-grade security, while supporting majority of LoRaWAN frequency bands.                                                                                                                                                                                                                                                                                                            
```

nejraselimovic commented on doc/content/gateways/COTX-Gateways_index.md:
```diff
@@ -0,0 +1,12 @@
+#COTX X1 Lite Gateway   
+#COTX X1 Gateway   
+#COTX X3 Gateway   
+#COTX X5 Gateway   
```
```suggestion
In the following subsection, you can find the documentation on how to connect COTX X1 Lite gateway.
```

nejraselimovic replied:
I see you have the intention to add more docs here, but you haven't added them yet. Please update this part when you add instructions for other gateways.

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+COTX X1 Lite is a standard LoRaWAN indoor gateway and a Helium data-only hotspot.
+It is highly compatible and has a user-friendly interface.
+Rich in software tools, powerful back office management and technical support.
+Compact and exquisite appearance, easy to install and deploy.
```
```suggestion
COTX X1 Lite is a standard LoRaWAN indoor gateway, with a compact appearance, easy to install and deploy. It is highly compatible and has a user-friendly interface. It is rich in software tools, powerful back office management and technical support.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+It is highly compatible and has a user-friendly interface.
+Rich in software tools, powerful back office management and technical support.
+Compact and exquisite appearance, easy to install and deploy.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
```
Please, don't use such file names. Change this image name to "x1-lite.png" or something similar. Also, this is not the way that we add images to our doc. Instead, we use the `figure` shortcode. Read our [CONTRIBUTING to docs guide](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/CONTRIBUTING.md).

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+Compact and exquisite appearance, easy to install and deploy.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
+
+##Key Features 
```
```suggestion
## Key Features 
```

nejraselimovic replied:
If there is no this space, the docs don't render properly. Please apply this fix for all occurrences.

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+LoRa band support：CN470,EU868,US915,AS923,KR920,AU915.
+LoRa power support up to 27dbm.
+Built-in Ethernet. Supports WiFi-AP mode as well as LTE-CAT4 uplink connectivity for easy adaptation to different scenarios.
+Gateway management and configuration based on OPENWRT, supporting MQTT bridging mode for easy deployment of gateways and integration environments.
```
It's fine to have a list of key features, but then you have to make it a list. Add bullets (by using `-` at the beginning of every row) and remove `.` at the end.

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+1.1 Required Equipment 
+COTX X1 Lite Gateway
+Power source
+Internet ready
```
This part is redundant. We assume that the reader owns this gateway already and that there is power source and internet connection. You can only add a few lines explaining HOW to connect the gateway to the internet and power it up, otherwise, you can delete this.

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+Power source
+Internet ready
+
+##1.2 Gateway Setup 
```
```suggestion
## Gateway Setup 
```

nejraselimovic replied:
We don't enumerate sections.

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+Internet ready
+
+##1.2 Gateway Setup 
+Step 1: First  insert the SMA male connector of the antenna feeder into the LoRa®  Antenna port of the device and tighten it.
```
```suggestion
**Step 1**: First  insert the SMA male connector of the antenna feeder into the LoRa®  Antenna port of the device and tighten it.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+
+##1.2 Gateway Setup 
+Step 1: First  insert the SMA male connector of the antenna feeder into the LoRa®  Antenna port of the device and tighten it.
+![Lite-3.png](./TTN-X1%20Lite-3.png)
```
same as explained above, change the file name to something simpler and add the image using the `figure` tag

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+Step 1: First  insert the SMA male connector of the antenna feeder into the LoRa®  Antenna port of the device and tighten it.
+![Lite-3.png](./TTN-X1%20Lite-3.png)
+
+Step 2: Place  your gateway to a suitable location (eg. near a window) to provide the best coverage for devices. 
```
```suggestion
**Step 2**: Place  your gateway to a suitable location (e.g. near a window) to provide the best coverage for devices. 
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+
+Step 2: Place  your gateway to a suitable location (eg. near a window) to provide the best coverage for devices. 
+
+Step 3: If you are using a wired Ethernet connection, connect an Ethernet cable between a router or switch and the Ethernet Port on the gateway，so the gateway can connect to the internet.
```
```suggestion
**Step 3**: If you are using a wired Ethernet connection, connect an Ethernet cable between a router or switch and the Ethernet Port on the gateway，so the gateway can connect to the internet.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+Step 2: Place  your gateway to a suitable location (eg. near a window) to provide the best coverage for devices. 
+
+Step 3: If you are using a wired Ethernet connection, connect an Ethernet cable between a router or switch and the Ethernet Port on the gateway，so the gateway can connect to the internet.
+![Lite-2.png](./TTN-X1%20Lite-2.png)
```
file name and `figure` tag

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+Step 3: If you are using a wired Ethernet connection, connect an Ethernet cable between a router or switch and the Ethernet Port on the gateway，so the gateway can connect to the internet.
+![Lite-2.png](./TTN-X1%20Lite-2.png)
+
+Step 4: Finally Insert the power adapter into the 110~220VAC mains socket, connect the  power supply into the PWR INPUT on the gateway.
```
```suggestion
**Step 4**: Finally Insert the power adapter into the 110~220 V AC mains socket, and connect the power supply into the PWR INPUT of the gateway.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+![Lite-2.png](./TTN-X1%20Lite-2.png)
+
+Step 4: Finally Insert the power adapter into the 110~220VAC mains socket, connect the  power supply into the PWR INPUT on the gateway.
+After plugging in the power, If the device indicator is green, the system is running properly. The Gateway should now connect to The Things Network .
```
```suggestion

After powering the gateway, you'll see a green light indicator if the system is running properly. The gateway should now be able to connect to {{% tts %}}.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+
+Step 4: Finally Insert the power adapter into the 110~220VAC mains socket, connect the  power supply into the PWR INPUT on the gateway.
+After plugging in the power, If the device indicator is green, the system is running properly. The Gateway should now connect to The Things Network .
+1.3 Registering the Gateway on TTN 
```
```suggestion

## Registering the Gateway on {{% tts %}} 
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+After plugging in the power, If the device indicator is green, the system is running properly. The Gateway should now connect to The Things Network .
+1.3 Registering the Gateway on TTN 
+1.Create a new {{% tts %}} account or login to your previously created {{% tts %}} account.
+2.Select CONSOLE for Europe1, North America1, or Australia1.
```
```suggestion
First, create a new {{% tts %}} account or login to your previously created {{% tts %}} account.

Navigate to {{% tts %}} Console depending on your region.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+1.3 Registering the Gateway on TTN 
+1.Create a new {{% tts %}} account or login to your previously created {{% tts %}} account.
+2.Select CONSOLE for Europe1, North America1, or Australia1.
+![Lite-4.png](./TTN-X1%20Lite-4.png)
```
file name and `figure` tag

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+1.Create a new {{% tts %}} account or login to your previously created {{% tts %}} account.
+2.Select CONSOLE for Europe1, North America1, or Australia1.
+![Lite-4.png](./TTN-X1%20Lite-4.png)
+3.Select Gateways , and then select Register Gateway.
```
```suggestion
Select **Gateways** and then select **Register gateway**.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+2.Select CONSOLE for Europe1, North America1, or Australia1.
+![Lite-4.png](./TTN-X1%20Lite-4.png)
+3.Select Gateways , and then select Register Gateway.
+4.Under Gateway ID, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”.
```
```suggestion

Under **Gateway ID**, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named `ID`.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+![Lite-4.png](./TTN-X1%20Lite-4.png)
+3.Select Gateways , and then select Register Gateway.
+4.Under Gateway ID, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”.
+5.Under Gateway EUI, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”
```
```suggestion

Under **Gateway EUI**, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named `ID`.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+3.Select Gateways , and then select Register Gateway.
+4.Under Gateway ID, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”.
+5.Under Gateway EUI, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”
+6.Under Description add a human readable description of the gateway.
```
```suggestion

Under **Description** add a human readable description of the gateway.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+4.Under Gateway ID, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”.
+5.Under Gateway EUI, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”
+6.Under Description add a human readable description of the gateway.
+7.Select a frequency plan from the dropdown box that corresponds to the frequency band supported by the Gateway.
```
```suggestion
Select a frequency plan that the gateway supports from the **Frequency Plan** drop-down list.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+5.Under Gateway EUI, enter the 8 byte gateway ID recorded on the bottom of gateway’s label named “ID”
+6.Under Description add a human readable description of the gateway.
+7.Select a frequency plan from the dropdown box that corresponds to the frequency band supported by the Gateway.
+8.The default connection of Gateway is Europe1, Gateway Server Address is: eu1.cloud.thethings.network, no additional other configuration. 
```
```suggestion
The default connection of gateway is to the `eu1.cloud.thethings.network` server, with no additional other configuration. If you're not in this region, you will have to change the Gateway Server address accordingly.
```

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+6.Under Description add a human readable description of the gateway.
+7.Select a frequency plan from the dropdown box that corresponds to the frequency band supported by the Gateway.
+8.The default connection of Gateway is Europe1, Gateway Server Address is: eu1.cloud.thethings.network, no additional other configuration. 
+![Lite-5.png](./TTN-X1%20Lite-5.png)
```
file name and `figure` tag

nejraselimovic commented on doc/content/gateways/COTX-X1 Lite gateway:
```diff
@@ -0,0 +1,47 @@
+7.Select a frequency plan from the dropdown box that corresponds to the frequency band supported by the Gateway.
+8.The default connection of Gateway is Europe1, Gateway Server Address is: eu1.cloud.thethings.network, no additional other configuration. 
+![Lite-5.png](./TTN-X1%20Lite-5.png)
+9.Click Create Gateway. 
```
```suggestion

Click **Register gateway**. 
```

nejraselimovic reviewed (changes requested):
This still doesn't render properly. Please apply changes that I suggested and request a review from me again.

Also, please organize these files. I suggest the following -> Create a folder named `cotx` in the `gateways` folder and a `cotx1` folder inside the `cotx` folder. Rename the `COTX-Gateways_index.md` file to `_index.md` and place it in the `cotx` folder. Rename the `COTX-X1 Lite gateway` file to `_index.md` file and place it in the `cotx1` folder. Rename all images as I suggested and place them in the `cotx1` folder. Feel free to ask if something is not clear, or just follow some other gateway's docs (you can check how other gateways' documentation was added here: https://github.com/TheThingsIndustries/lorawan-stack-docs/tree/master/doc/content)/gateways

FrankZhaoX committed:
Update doc/content/gateways/COTX-Gateways_index.md

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-Gateways_index.md

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-Gateways_index.md

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Update doc/content/gateways/COTX-X1 Lite gateway

Co-authored-by: Nejra Selimović <nejra.selimovic@thethingsindustries.com>

FrankZhaoX committed:
Move files from home directory to cotx folder

FrankZhaoX committed:
Move files from home directory to cotx folder

FrankZhaoX committed:
Rename COTX-X1 Lite ga to COTX-X1_Lite_gateway

FrankZhaoX committed:
Delete TTN-X1 Lite-1.png

FrankZhaoX committed:
Delete TTN-X1 Lite-2.png

FrankZhaoX committed:
Delete TTN-X1 Lite-3.png

FrankZhaoX committed:
Delete TTN-X1 Lite-4.png

FrankZhaoX committed:
Merge pull request #9 from FrankZhaoX/FrankZhaoX-patch-8

Delete TTN-X1 Lite-4.png

FrankZhaoX committed:
Delete TTN-X1 Lite-2.png

FrankZhaoX committed:
Merge pull request #10 from FrankZhaoX/FrankZhaoX-patch-8

Delete TTN-X1 Lite-2.png

FrankZhaoX committed:
Merge pull request #8 from FrankZhaoX/FrankZhaoX-patch-7

Delete TTN-X1 Lite-3.png

FrankZhaoX committed:
Merge pull request #7 from FrankZhaoX/FrankZhaoX-patch-6

Delete TTN-X1 Lite-2.png

FrankZhaoX committed:
Merge pull request #6 from FrankZhaoX/FrankZhaoX-patch-5

Delete TTN-X1 Lite-1.png

FrankZhaoX committed:
Merge pull request #5 from FrankZhaoX/FrankZhaoX-patch-4

Rename COTX-X1 Lite ga to COTX-X1_Lite_gateway

FrankZhaoX committed:
Merge pull request #4 from FrankZhaoX/FrankZhaoX-patch-3

Move files from home directory to cotx folder

FrankZhaoX committed:
Merge pull request #3 from FrankZhaoX/FrankZhaoX-patch-2

Move files from home directory to cotx folder

FrankZhaoX committed:
Delete TTN-X1 Lite-5.png

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+---
+
+COTX X1 Lite is a standard LoRaWAN indoor gateway, with a compact appearance, easy to install and deploy. It is highly compatible and has a user-friendly interface. It is rich in software tools, powerful back office management and technical support.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
```
Please use the `figure` tag to insert images. If you're not sure how, read the documentation contributing guidelines in this repo, or check .md files of other gateways for an example. This applies to all images in this PR.

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+COTX X1 Lite is a standard LoRaWAN indoor gateway, with a compact appearance, easy to install and deploy. It is highly compatible and has a user-friendly interface. It is rich in software tools, powerful back office management and technical support.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
+
+## Key Features 
```
```suggestion
## Key Features 

```

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+Built-in Ethernet. Supports WiFi-AP mode as well as LTE-CAT4 uplink connectivity for easy adaptation to different scenarios.
+Gateway management and configuration based on OPENWRT, supporting MQTT bridging mode for easy deployment of gateways and integration environments.
+
+##Provisioning on TTN Network Server 
```
```suggestion
## Provisioning on {{% tts %}} Network Server

```

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+Gateway management and configuration based on OPENWRT, supporting MQTT bridging mode for easy deployment of gateways and integration environments.
+
+##Provisioning on TTN Network Server 
+1.1 Required Equipment 
```
Remove enumeration

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+
+##Provisioning on TTN Network Server 
+1.1 Required Equipment 
+COTX X1 Lite Gateway
```
Add bullets

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+Power source
+Internet ready
+
+## Gateway Setup 
```
```suggestion
## Gateway Setup

```

nejraselimovic commented on doc/content/gateways/cotx/COTX-X1_Lite_gateway:
```diff
@@ -0,0 +1,51 @@
+
+After powering the gateway, you'll see a green light indicator if the system is running properly. The gateway should now be able to connect to {{% tts %}}.
+
+## Registering the Gateway on {{% tts %}} 
```
```suggestion
## Registering the Gateway on {{% tts %}}

```

nejraselimovic reviewed (changes requested):
I still see some points from my feedback that aren't addressed properly, for example enumeration (1.1, 1.2 etc.) and the way of inserting images, so I added some comments again. Please fix those and squash these commits into a single commit, so I can merge this later.

FrankZhaoX committed:
COTX Gateways

COTX Gateways

FrankZhaoX committed:
Merge branch 'TheThingsNetwork:master' into master

FrankZhaoX committed:
Delete COTX Gateways directory

FrankZhaoX committed:
Add files via upload

FrankZhaoX committed:
Merge pull request #11 from FrankZhaoX/FrankZhaoX-patch-2

Add files via upload

nejraselimovic commented:
Since my feedback hasn't been fully addressed and this PR is pushed to a wrong repository (just spotted that), I'm closing this PR, adding my changes and replacing it with https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/954.
